### PR TITLE
Add welcome and sign-up frontend

### PR DESCRIPTION
**Pull request by PerezIgnacio**
This pull request:
- Changes .erb files to .haml
- Adds content to welcome view
- Adds styles to welcome and sign-up views

### CODE REVIEW

File: app/views/welcome/index.html.haml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+= render layout: 'layouts/main' do
+  = image_tag 'smilies.png', :srcset => {'smilies@2x.png' => '2x', 'smilies@3x.png' => '3x'}, |
+    :class=>'img-smilies'                                                                     |
+  .welcome-title TARGET MVD
```
Comment (mdornel) [inline app/views/welcome/index.html.haml]: Use [I18n](https://edgeguides.rubyonrails.org/i18n.html) as a dictionary to store this kind of text

File: app/views/welcome/index.html.haml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  .welcome-subtitle Find people near you  Connect
```
Comment (jpascual1994) [inline app/views/welcome/index.html.haml]: Please use `I18n`

File: app/views/devise/registrations/new.html.haml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+= render layout: 'layouts/main' do
+  .sign-up-title SIGN UP
+  = form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
+    = render 'devise/shared/error_messages', resource: resource
+    .sign-up-field
+      = f.label :name
+      %br/
+      = f.text_field :name, autofocus: true, |
+        :class => ('error-field' if resource.errors.include? :name)
```
Comment (mdornel) [inline app/views/devise/registrations/new.html.haml]: use newer syntax for hash: `class: ('error-field' if resource.errors.include? :name)`

Comment (mdornel) [reply app/views/devise/registrations/new.html.haml]: Replace it in other places below

File: app/assets/stylesheets/logged_out.scss [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  height: 100%;
+  overflow-x: hidden;
+  position: fixed;
+  top: 0;
+  width: 50%;
+  z-index: 1;
+}
+
+.left {
+  left: 0;
```
Comment (jpascual1994) [inline app/assets/stylesheets/logged_out.scss]: Use only one line if you are defining only one property
`.left { left: 0; }`

File: config/locales/en.yml [hunk 5]
```diff
@@ -30,4 +30,19 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  assets:
+    images:
```
Comment (jpascual1994) [inline config/locales/en.yml]: Remove the files names from this file and write the file names directly on the views.

**Pull request merged**